python/cdk_constructs/stack.py

```diff
@@ -18,8 +17,6 @@ def __init__(
         Currently contains:
             API stack containing a full rest API with lambda
```

Comment:
I'm not sure I follow why it's called `Stack` exactly? Is it a stack of the results of the API calls? Why's this needed exactly?

just wondering

Reply:
It's just a cloud term - AWS chose it pretty randomly. I think it's a stack of cloud resources, with App being the route and then every function gets added to the stack when I pass it or an nested resources as the "scope" argument.

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacks.html

I can update the documentation